shared/src/test/diff/typegen/TypegenTerms.mls

```diff
@@ -0,0 +1,156 @@
+:ts
+:ne
```

Comment:
If you never want to execute the code, you could just add `:NoJS` at the beginning of the file.